app.py

```diff
@@ -35,9 +39,11 @@ def make_pdf_from_url():
     pdf_file = make_pdf(source)
     return send_file(pdf_file, as_attachment=False)
 
+cross_origin(["www.pawsquad.com", "martinwork.eu.ngrok.io"])
```

Comment:
Can you remove these hard coded domains and put into env vars?

Reply:
Maybe something like this... 
```
# heroku config:set CORS_DOMAINS="www.pawsquad.com,martinwork.eu.ngrok.ido"
from os import environ as env
cross_origin(env.get('CORS_DOMAINS', '').split(','))
```

Mind putting something in the README about how to use the cors code you've added?

Thanks